posts/2021-11-01-Rust-1.56.1.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: "Announcing Rust 1.56.1"
+author: The Rust Release Tea
```

Comment:
```suggestion
author: The Rust Release Team
```

Reply:
This was actually supposed to be the security response wg :sweat_smile: Dunno what happened, fixed.